invenio_webhooks/models.py

```diff
@@ -176,7 +176,7 @@ def status(self, event):
         return (
             self.CELERY_STATES_TO_HTTP.get(result.state),
             result.info.get('message')
-            if result.state == states.PENDING and result.info
+            if result.state in [states.PENDING, states.STARTED] and result.info
```

Comment:
`in {states.PENDING, states.STARTED}` for constant lookup times

plus you can move it as class property `CELERY_RESULT_INFO_FOR = {states.PENDING, states.STARTED}` so if you really want to change the behavior for specific receiver you can override it easily.